packages/@azure/servicebus/data-plane/lib/topicClient.ts

```diff
@@ -129,4 +134,15 @@ export class TopicClient implements Client {
       throw new Error("The topicClient has been closed and can no longer be used.");
     }
   }
+
+  /**
+   * Returns the corresponding dead letter topic name for the given topic and subscription names.
+   * Use this in the `createSubscriptionClient` function of the `ServiceBusClient` instance to
+   * receive messages from dead letter queue for given subscription.
+   * @param topicName
+   * @param subscriptionName
+   */
+  static getDeadLetterTopicPath(topicName: string, subscriptionName: string): string {
```

Comment:
Should we get rid of this and instead have a method that gives the user back a receiver for the dead letter topic? Same for the other one?

Reply:
Covered in https://github.com/Azure/azure-sdk-for-js/issues/1424